docs/run-platform-server/ingestion.mdx

```diff
@@ -5,26 +5,35 @@ sidebar_position: 45
 
 import { CodeExample } from "@site/src/components/CodeExample";
 
-Horizon provides access to both current and historical state on the Stellar network through a process called **ingestion**.
-
-Horizon provides most of its utility through ingested data, and your Horizon server can be configured to listen for and ingest transaction results from the Stellar network. Ingestion enables API access to both current (e.g. someone's balance) and historical state (e.g. someone's transaction history).
+Horizon API provides most of its utility through ingested data, and your Horizon server can be configured to listen for and ingest transaction results from the Stellar network. Ingestion enables API access to both current state (e.g. someone's balance) and historical state (e.g. someone's transaction history).
 
 ## Ingestion Types
 
 There are two primary ingestion use-cases for Horizon operations:
 
-- ingesting **live** data to stay up to date with the latest, real-time changes to the Stellar network, and
-- ingesting **historical** data to peek how the Stellar ledger has changed over time
+- ingesting **live** data to stay up to date with the latest ledgers from the network, accumulating a sliding window of aged ledgers
+- ingesting **historical** data to retroactively add network data from a time range in the past to the database
+
+## Determine storage space
+
+You should think carefully about the historical timeframe of ingested data you'd like to retain in Horizon's database. The storage requirements for transactions on the Stellar network are substantial and are growing unbounded over time. This is something that you may need to continually monitor and reevaluate as the network continues to grow. We have found that most organizations need only a small fraction of recent historical data to satisfy their use cases. Through analyzing traffic patterns on SDF's Horizon instance, we see that most requests are for very recent data.
+
+To keep your storage footprint small, we recommend the following:
+
+- use **live** ingestion only use and depend on **historical** ingestion in limited exceptional cases
+- if your application requires access to all network data, no filtering can be done, we recommend limiting historical retention of ingested data to a sliding window of 1 month (HISTORY_RETENTION_COUNT=518400) which is default set by Horizon
```

Comment:
Since this is the first time we're mentioning "filtering" in this context, it might be helpful to add link to the ingestion-filtering doc.